cmd/exec.go

```diff
@@ -32,10 +41,32 @@ var execCmd = &cobra.Command{
 		return nil
 	},
 	RunE: execRun,
+	Example: `
+Given a secret store like this:
+
+	$ echo '{"db_username": "root", "db_password": "hunter22"}' | chamber import - 
+
+--strict will fail with unfilled env vars
+
+	$ HOME=/tmp DB_USERNAME=chamberme DB_PASSWORD=chamberme EXTRA=chamberme chamber exec --strict service exec -- env
+	chamber: extra unfilled env var EXTRA
+	exit 1
+
+--pristine takes effect after checking for --strict values
+
+	$ HOME=/tmp DB_USERNAME=chamberme DB_PASSWORD=chamberme chamber exec --strict --pristine service exec -- env
```

Comment:
Is it worth logging here that we're using pristine? In case someone is debugging and doesn't realize this is enabled 😄

Reply:
Done in https://github.com/segmentio/chamber/pull/227/commits/e6bf6b50580c81f186d8131469ea9bbbd0b1d49b. But only for verbose mode. Considering we're execing stuff here, users probably generally don't want us cluttering up the real app's stderr.